demos/storybook/stories/drawer/with-full-config.tsx

```diff
@@ -176,45 +289,28 @@ export const withFullConfig = (context: DrawerStoryContext): StoryFnReactReturnT
             },
             icon: <NotificationsActive />,
         },
-    ].slice(0, numberLinksGroup2);
+    ];
 
     // Footer
-    const showFooter = boolean('show footer', true, footerGroupId);
+    const showFooter = boolean('Show footer', true, footerGroupId);
     const footerBackgroundColor = color('backgroundColor', Colors.white[50], footerGroupId);
 
     return (
-        <Drawer open={open} width={width} key={'drawer'}>
-            <DrawerHeader
-                title={headerTitle}
-                subtitle={headerSubtitle}
-                icon={headerIcon}
-                backgroundImage={headerBackgroundImage}
-                fontColor={headerFontColor}
-                backgroundColor={headerBackgroundColor}
-            />
-
-            <DrawerBody
-                iconColor={bodyIconColor}
-                fontColor={bodyFontColor}
-                backgroundColor={bodyBackgroundColor}
-                activeFontColor={bodyActiveFontColor}
-                activeBackgroundColor={bodyActiveBackgroundColor}
-                activeIconColor={bodyActiveIconColor}
-                chevron={bodyChevron}
-            >
-                <DrawerNavGroup
-                    activeItem={context.state.selected}
-                    divider={bodyDividers}
-                    items={links1}
-                    title={groupTitle1}
-                />
+        <Drawer key={'drawer'} activeItem={context.state.selected} {...drawerKnobs}>
+            <DrawerHeader {...headerKnobs} />
+            <DrawerBody {...bodyKnobs}>
```

Comment:
I'm conflicted on this change. I like the idea of having all of the knobs defined together in an object, but now the code snippets in the Story tab are not very useful.

Reply:
They are still useful, just stacked together rather than listed out individually.

Reply:
But if a user wants to copy-paste the snippet, they have no idea what any of the props actually are unless they go digging around in the source:
![image](https://user-images.githubusercontent.com/29152776/76850203-476eb780-681d-11ea-9f2f-fa5fd0a2270a.png)

Reply:
@emclaug2 what do you think about this?